### PR TITLE
add trigger facebook_quick_reply 

### PR DESCRIPTION
**Pull request by afbayonac**
#992

### CODE REVIEW

**Comment by peterswimm** [comment]
There are some changes that are broken with the latest push, can you review the conflicting files and provide some background on the changes you are making for review?

File: lib/Facebook.js [hunk 1]
```diff
@@ -315,6 +315,23 @@ function Facebookbot(configuration) {
...
 
+                    } else if (facebook_message.message && facebook_message.message.quick_reply) {
+
+                        // trigger BOTH a quick_reply event
+                        // and a normal message received event.
+                        // this allows developers to receive quick_reply as part of a conversation.
+                        var message = {
+                            text: facebook_message.message.quick_reply.payload,
+                            payload: facebook_message.message.quick_reply.payload,
+                            input: facebook_message.message.text,
```
Comment (ouadie-lahdioui) [inline lib/Facebook.js]: Why are you added an input entry to the message ?

**Comment by afbayonac** [comment]
ready !!

**Comment by benbrown** [comment]
This is interesting! 

I agree with @ouadie-lahdioui here - this would be a breaking change for a lot of old bots that expect quick replies to work like normal text.

This could be achieved with an _optional_ categorize middleware that looks for the quick_replies field and changes the message type.

Thoughts?

**Comment by benbrown** [comment]
We are currently not going to change the way quick replies are handled.

**Pull request closed without merging**